native/nativescript/README.md

```diff
@@ -1 +1,6 @@
-# NativeScript
\ No newline at end of file
+# NativeScript
+
+To avoid Out Of Memory exception:
+1- Try as possible use small images size.
```

Comment:
Can we write this as `Try when possible to use...`?

Reply:
Sure, Thanks for that suggestion.